src/bundle/createSES.js

```diff
@@ -90,7 +90,7 @@ export function createSESWithRealmConstructor(creatorStrings, Realm) {
     // Build a harden() with an empty fringe. It will be populated later when
     // we call harden(allIntrinsics).
     const makeHardenerSrc = `(${makeHardener})`;
-    const harden = r.evaluate(makeHardenerSrc)();
+    const harden = r.evaluate(makeHardenerSrc)(options.fringeSet);
```

Comment:
If we resolve the other PR according to https://github.com/Agoric/make-hardener/pull/35#pullrequestreview-225857583 , then this will need the obvious change.

Reply:
I will withdraw options.fringeSet for now until SES uses a version of make-hardener that has the necessary support so I can write tests against it.

I can test the whitelist changes now, so I will add them to this PR.